array-manipulation-1/src/initial.ts

```diff
@@ -0,0 +1,10 @@
+/* exported initial */
+function initial(array: unknown[]): unknown[] {
+  const initialArr = [];
+  for (let i = 0; i < array.length; i++) {
+    if (i !== array.length - 1) {
```

Comment:
While this does work, instead we can just set `i < array.length - 1`. We are able to use the result of an expression in our conditional expression.